src/live.asp.net/wwwroot/css/site.css

```diff
@@ -137,6 +137,10 @@ footer {
         font-size: 0.8em;
     }
 
+.label-new-show{
```

Comment:
Sorry again, I'm working with many branches and there's sort of overlapping
thanks again